manim/scene/scene_file_writer.py

```diff
@@ -435,34 +462,27 @@ def combine_movie_files(self):
         # cuts at all the places you might want.  But for viewing
         # the scene as a whole, one of course wants to see it as a
         # single piece.
-        kwargs = {
-            "remove_non_integer_files": True,
-            "extension": file_writer_config["movie_file_extension"],
-        }
-        if file_writer_config["from_animation_number"] is not None:
-            kwargs["min_index"] = file_writer_config["from_animation_number"]
-        if file_writer_config["upto_animation_number"] not in [None, np.inf]:
-            kwargs["max_index"] = file_writer_config["upto_animation_number"]
-        else:
-            kwargs["remove_indices_greater_than"] = self.scene.num_plays - 1
```

Comment:
Question: where did these go?

Reply:
This has been removed. 
This is now depreceted as the system with integreerds index names for partial movie files (ie 0001.mp4, 0002.mp4) does not exist in a post 166 world. 
As `kwargs["min_index"]` and all this stuff was used to handle the files named with that these names, it's now useless.

Reply:
Wait so the names of the files in partial_movie_files are changing too?